From cda73b54642ab23ea32f95866a8ed3295f6a32e7 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:45:11 +0100
Subject: [PATCH 2/2] Squash of
 github/lineage-16.0(05ab40a1)..lineage-16.0-android-9.0.0_r31(8e85f29f) (DO
 NOT SUBMIT)

Hector Dearman (1):
 - Make system_server atrace category work with traced_probes

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (3):
 - Snap for 5053079 from 63d07d75868e56f3fd40a4a1747b5af09528451a to
   pi-qpr2-release
 - Snap for 5053079 from 63d07d75868e56f3fd40a4a1747b5af09528451a to
   pi-qpr2-release
 - Merge "Snap for 5053079 from 63d07d75868e56f3fd40a4a1747b5af09528451a
   to pi-qpr2-release" into pi-qpr2-release

Change-Id: I9226574e09b5ab33b7101ca7c03bec1a15f6a48d

Change-Id: I578b66519bcf088978d0327f2e41b2887585adfc
---
 prebuilts/api/28.0/private/atrace.te | 2 ++
 private/atrace.te                    | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/prebuilts/api/28.0/private/atrace.te b/prebuilts/api/28.0/private/atrace.te
index 630935da..1b86d3e1 100644
--- a/prebuilts/api/28.0/private/atrace.te
+++ b/prebuilts/api/28.0/private/atrace.te
@@ -22,6 +22,8 @@ set_prop(atrace, debug_prop)
 binder_use(atrace)
 allow atrace healthd:binder call;
 allow atrace surfaceflinger:binder call;
+allow atrace system_server:binder call;
+
 get_prop(atrace, hwservicemanager_prop)
 
 allow atrace {
diff --git a/private/atrace.te b/private/atrace.te
index 630935da..1b86d3e1 100644
--- a/private/atrace.te
+++ b/private/atrace.te
@@ -22,6 +22,8 @@ set_prop(atrace, debug_prop)
 binder_use(atrace)
 allow atrace healthd:binder call;
 allow atrace surfaceflinger:binder call;
+allow atrace system_server:binder call;
+
 get_prop(atrace, hwservicemanager_prop)
 
 allow atrace {
-- 
2.17.1

